### ISSUE TITLE
Create a README.md file

### ISSUE THREAD

**Issue by JayKandari** [gunjana/migrationtodrupal#2]
Please create a README.md file which shows how to use this module. Mention all the things that this module does. If given step by step even better.

**Comment by gunjana** [comment]
Created README.md with steps to configure

**Comment by JayKandari** [comment]
![image](https://github.com/gunjana/migrationtodrupal/assets/1266231/bdbff275-01d3-47d6-adf7-e6b86ff7f62d)
The readme file still doesn't has few key points. Eg: 

How to use this module. 
How to run migration/rollback etc. 
From where the data is coming 
How can fields be mapped using the config form etc. 
Add more details if you like. 

Thanks!

**Comment by gunjana** [comment]
Hi @JayKandari  this is not the README.md please refer to the file inside the module folder
https://github.com/gunjana/migrationtodrupal/blob/main/migration_to_drupal/README.md

**Comment by JayKandari** [comment]
I see. Thank you for pointing to the correct README file. 

I'll keep this ticket open and will close together with another ticket where it mentions about mongodb. as those instructions also need to be updated onto this README file once that is finished. 

Also a small nitpick - let us use proper markdown syntax in .md files for better readability and formatting. 

Thanks again!

**Comment by JayKandari** [comment]
Closing this ticket... most of the changse mentioned are done.. 
![image](https://github.com/gunjana/migrationtodrupal/assets/1266231/6dcd0ded-7799-49a1-9dd0-79697c48b1a8)
